kfs-core/src/main/java/edu/arizona/kfs/fp/document/ProcurementCardDocument.java

```diff
@@ -93,4 +153,36 @@ protected String buildDocumentTitle(String title) {
          
         return title;
     }
-}
\ No newline at end of file
+    
+    protected boolean allowBackpost(Date tranDate) {
+        ParameterService parameterService = SpringContext.getBean(ParameterService.class);
+        UniversityDateService universityDateService = SpringContext.getBean(UniversityDateService.class);
+       
+        int allowBackpost = (Integer.parseInt(parameterService.getParameterValueAsString(ProcurementCardLoadStep.class, ALLOW_BACKPOST_DAYS)));
```

Comment:
Please separate this in two steps:
String allowBackpostParam = parameterService.getParameter.....
int allowBackpost = 0; -> also please check if this is an OK default value  
if (StringUtils.isNotEmpty() && StringUtils.isNumeric( allowBackPostDyas ) ){
    allowBackpost = Integer.parseInt( allowBackpostParam );
}